src/wtc-gallery-component.js

```diff
@@ -52,10 +73,67 @@ class Gallery extends ElementController {
       this.element.appendChild(this.prevBtn);
     }
 
-    // Add pause-on-hover mouse events:
+    // If pagination is set to true, set up the item list
+    if (this.options.pagination) {
+
+      // if a nodeList was provided, use it.
+      // otherwise, build a generic list of buttons
+      if (this.options.paginationTarget) {
+
+        let itemList = this.options.paginationTarget,
+          items = itemList.querySelectorAll('li');
```

Comment:
I think you're right @marlonmarcello. Better to let the developer decide what kind of element to use, rather than being opinionated about a `<ul>`.

I prefer your second suggestion, about querying for the children of whatever you pass in. This way there's no need to remember a specific class name or anything.